SwiftSyntaxCLI/Sources/swift-syntax-cli/common/Paths.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  File.swift
+//
+//
+//  Created by Kim de Vos on 26/04/2023.
+//
+
+import Foundation
+
+enum Paths {
+  static var modules: [String] {
+    ["SwiftBasicFormat", "SwiftIDEUtils", "SwiftParser", "SwiftSyntax", "SwiftSyntaxBuilder"]
+  }
```

Comment:
Technically, these aren’t paths, right? Also, this doesn’t include all modules. Given that you’re only using it in `VerifySourceCode`, I think it should just be specified there.